<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmegaWindow</class>
 <widget class="QMainWindow" name="OmegaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Omega Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Media/omega.png</normaloff>:/Media/omega.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QOmgTreeEntities" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Entities</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>400</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>248</width>
             <height>278</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="bigScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>298</width>
           <height>566</height>
          </rect>
         </property>
         <zorder>renderWidget</zorder>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="renderWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QOmgDragableEntities" name="entities" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{border:1px solid black;}

QLabel{border:1px soid white;}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="_fps_label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>FPS:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="_p_tree_entities">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Tree Entities</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Media/tree_folder.png</normaloff>:/Media/tree_folder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="OmegaToolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOmega"/>
   <addaction name="actionSave"/>
   <addaction name="actionExport_To_Omg"/>
   <addaction name="actionImport_Omg"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOmega"/>
    <addaction name="actionExport_To_Omg"/>
    <addaction name="actionImport_Omg"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Folder"/>
    <addaction name="actionNewGame"/>
   </widget>
   <widget class="QMenu" name="menuEntities">
    <property name="title">
     <string>Entities</string>
    </property>
    <addaction name="actionNew_Player"/>
    <addaction name="actionNew_Enemy"/>
    <addaction name="actionNew_Scene_Object"/>
    <addaction name="actionNew_Item"/>
    <addaction name="actionNew_Weapon"/>
    <addaction name="actionNew_Map"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEntities"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Player"/>
   <addaction name="actionNew_Enemy"/>
   <addaction name="actionNew_Scene_Object"/>
   <addaction name="actionNew_Item"/>
   <addaction name="actionNew_Weapon"/>
   <addaction name="actionNew_Map"/>
   <addaction name="actionNew_Folder"/>
   <addaction name="actionNewGame"/>
  </widget>
  <action name="actionOmega">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/omega.png</normaloff>:/Media/omega.png</iconset>
   </property>
   <property name="text">
    <string>Omega</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/exit.png</normaloff>:/Media/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNew_Player">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/character.png</normaloff>:/Media/character.png</iconset>
   </property>
   <property name="text">
    <string>New Player</string>
   </property>
  </action>
  <action name="actionNew_Enemy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/enemy.png</normaloff>:/Media/enemy.png</iconset>
   </property>
   <property name="text">
    <string>New Enemy</string>
   </property>
  </action>
  <action name="actionNew_Scene_Object">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/sceneobj.png</normaloff>:/Media/sceneobj.png</iconset>
   </property>
   <property name="text">
    <string>New Scene Object</string>
   </property>
  </action>
  <action name="actionNew_Item">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/item.png</normaloff>:/Media/item.png</iconset>
   </property>
   <property name="text">
    <string>New Item</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/save.png</normaloff>:/Media/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew_Weapon">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/gun.png</normaloff>:/Media/gun.png</iconset>
   </property>
   <property name="text">
    <string>New Weapon</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/folder.png</normaloff>:/Media/folder.png</iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionExport_To_Omg">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/export.png</normaloff>:/Media/export.png</iconset>
   </property>
   <property name="text">
    <string>Export To Omg</string>
   </property>
  </action>
  <action name="actionImport_Omg">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/import.png</normaloff>:/Media/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Omg</string>
   </property>
  </action>
  <action name="actionNew_Map">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/map.png</normaloff>:/Media/map.png</iconset>
   </property>
   <property name="text">
    <string>New Map</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Media/game.png</normaloff>:/Media/game.png</iconset>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QOmgTreeEntities</class>
   <extends>QTreeWidget</extends>
   <header>QtComponents/QOmgTreeEntities.h</header>
  </customwidget>
  <customwidget>
   <class>QOmgDragableEntities</class>
   <extends>QWidget</extends>
   <header>QtComponents/QOmgDragableEntities.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
